### ISSUE TITLE
Mute all fav/boost notifications for specific toot

### ISSUE THREAD

**Issue by melissaboiko** [mastodon/mastodon#22860]
### Pitch

I would love a feature to clear the notification feed from all the likes and boosts for a particular toot, past and future, in the same vein as "hide conversation".

### Motivation

With the influx of users coming from the Twitter collapse, I now experience my toots going viral and getting hundreds of likes/boosts.   Only after this happened, I realise that I treasured the notification feed because it would show me reactions by friends in my clique (people I know personally, or people whom I recognise their online personas and personalities).

It is much more important to me to know which 6 friends liked that low-effort selfie, than to know which 300 strangers liked the well-timed hot take.   Right now it is effectively impossible for me to know which of my friends reacted to which of my toots, because any human-scale information is buried in an endless stream of favs and boosts by strangers.

A feature like this would help in keeping Mastodon be a human-scale, community-building tool rather than becoming another addictive chase after clout/big numbers.

Related issues:
 - Digital well-being by default #20186
 -  Hide a specific notification #1900
 - Hide a specific toot #1669

**Comment by michealmikeyb** [comment]
future notifications are handled by muting a conversation , Not sure if we want to make the default behavior of muting to delete past notifications so created a PR to add a clear notifications button

**Comment by RockstarRaccoon** [comment]
Agreed: if you have a post go viral, and this has happened to me, your notifications just become a constant dinging as they're flooded by the one thing until you turn them off for everything.  This would be good for people who would like to see boosts and faves, but post that one meme or whatever that travels across the fediverse in record time.

**Comment by melissaboiko** [comment]
@michealmikeyb Does "mute conversation" mutes favs and boosts notifications too, or just replies? If the former I would suggest rephrasing it to "mute notifications"; to me at least, "mute conversation" sounds like it will mute replies only.

A button for "clear all notifications for toot" plus mute would be a solution, yes.  Another possibility would be to collapse like/reboosts on your notification feed (so you'd see a single notification saying "300 people fav'ed this toot", rather than 300 notifications of "account X fav'ed this toot").

**Comment by michealmikeyb** [comment]
@melissaboiko from my testing mute conversation does stop incoming boost and fave notifications. Could change the verbage on it but it should probably be just as concise as the menu is small especially on mobile, don't want to go past 2 words.

I like the idea of collapsible notifications but I'm a newbie to the project and that's beyond my u.i. skills.

**Comment by eishiya** [comment]
If I understand correctly that this is about muting fav/boost notifications for a given post without muting notifications for *replies* to that post, then I'd like to support this. I often want to mute favs/boosts for my posts, but I still want to know when people post actual responses.
